pkg/scorecard.go

```diff
@@ -92,9 +92,17 @@ func RunScorecards(ctx context.Context,
 		return ScorecardResult{}, err
 	}
 
+	commits, err := repoClient.ListCommits()
+	if err != nil {
+		//nolint:wrapcheck
+		return ScorecardResult{}, err
+	}
+
+	// There is always at least one commit.
```

Comment:
I don't think we should make this assumption. I wouldn't be surprised if one of the repos in our cron jobs breaks due to this.